drizzlepac/haputils/catalog_utils.py

```diff
@@ -974,6 +974,13 @@ def identify_sources(self, **pars):
                                                               source_box=self._size_source_box,
                                                               mask=mask)
 
+            # Check if custom_segm_image is None indicating there are no detectable sources in the 
+            # total detection image.  If value is None, a warning has already been issued.  Issue
+            # a final message and return.
+            if custom_segm_img is None:
+                log.warning("End processing for the Segmentation Catalog due to no sources detected with Custom or Gaussian kernel.")
+                return None
```

Comment:
It is not clear that this needs to return any value at all, since this method was not designed to return a value upon success.  A simple empty return statement should be sufficient here.  Same for Line 1027 as well.

Reply:
Remove unnecessary return value of None.